Sources/LocalReceiptParsing/BasicTypes/InAppPurchase.swift

```diff
@@ -36,6 +36,17 @@ extension AppleReceipt {
 
 }
 
+extension AppleReceipt.InAppPurchase {
+
+    var isActiveSubscription: Bool {
+        guard self.productType?.isSubscription == true else { return false }
+        guard let expiration = self.expiresDate else { return true }
```

Comment:
When would this happen?

Reply:
non subscriptions won't have expiration dates

Reply:
so in theory you'd never get here, but better safe than sorry